Controller/Api/MessageController.php

```diff
@@ -135,10 +149,17 @@ public function postMessageAction(Request $request)
             $this->messageManager->save($message);
 
             $view = FOSRestView::create($message);
-            $serializationContext = SerializationContext::create();
+
+            if (class_exists('FOS\RestBundle\Context\Context')) {
```

Comment:
Not sure if this condition is the right one after all, since it only fails for FOSRest 1… maybe you would be better off reusing the `property_exists` condition?